01_fibonacci/fib.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fibonacci(10)
+}
+
+func fibonacci(n int) {
+
+	var sum int
```

Comment:
`sum` must start from zero. You can express this intent with `sum := 0`. But actually: a) you should declare it where you set it inside the loop, and b) you don't really need it at all.

Reply:
yes, added as a local variable